properties.schema

```diff
@@ -144,42 +144,42 @@
             "type": "string",
             "inputType": "ColourPicker",
             "default": "#4D4D4D",
-            "help": "Question button background colour"
+            "help": "Defines component question buttons and navigational elements."
           },
           "btn-color-inverted": {
             "title": "Button colour - inverted",
             "type": "string",
             "inputType": "ColourPicker",
             "default": "#FFFFFF",
-            "help": "Question button foreground colour"
+            "help": "Defines the Btn text / icon colour. Should be a colour that provides good contrast against the Btn colour."
           },
           "btn-color-hover": {
             "title": "Button colour - hover",
             "type": "string",
             "inputType": "ColourPicker",
             "default": "",
-            "help": "Question button background colour when hovered over"
+            "help": "Defines the Btn colour when hovered over."
           },
           "btn-color-inverted-hover": {
             "title": "Button colour - inverted hover",
             "type": "string",
             "inputType": "ColourPicker",
             "default": "",
-            "help": "Question button foreground colour when hovered over"
+            "help": "Defines the Btn text / icon colour when hovered over. Should be a colour that provides good contrast against the Btn hover colour."
           },
           "disabled": {
             "title": "Disabled colour",
             "type": "string",
             "inputType": "ColourPicker",
             "default": "#DDDDDD",
-            "help": "Question button disabled background colour"
+            "help": "Defines the Btn colour when it is disabled."
```

Comment:
I'm not sure this help text really adds anything, the title seems pretty self-explanatory. Seems minor I know but it would save on i18n effort if left out.